sections/contact-form.liquid

```diff
@@ -16,7 +16,9 @@
 
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="contact page-width page-width--narrow section-{{ section.id }}-padding">
-    <h2 class="title title-wrapper--no-top-margin {{ section.settings.heading_size }}">{{ section.settings.heading | escape }}</h2>
+    {%- if section.settings.heading != blank -%}
+      <h2 class="title title-wrapper--no-top-margin {{ section.settings.heading_size }}">{{ section.settings.heading | escape }}</h2>
```

Comment:
> On "Contact Us" templates only, remove the empty h2 element from the DOM. Keep the heading for "Contact Us" sections. 

This `contact-form.liquid` file is used for the contact form section and the contact page template. 
Adding this condition will remove the heading for sections and for page templates. 

Something to consider is that there is no guarantee that a merchant will use the `Contact us` Title for the contact us page. The contact us template can also be assigned to multiple pages, other than the contact us page. 

We could consider using a hidden `h2` for sections when a merchant doesn't include a heading for the section. 

```
<h2 class="visually-hidden">Contact us</h2>
```

^ This _may_ be more verbose for a Contact us page assigned to the the contact us template. That said, this section can be added to other templates, unrelated to the contact us page, and it could be useful to add the hidden `h2` to make the purpose of the form clearer.

Reply:
> We could consider using a hidden `h2` for sections when a merchant doesn't include a heading for the section.

Agreed. As is the same solution suggested in https://github.com/Shopify/dawn/issues/1457.

Reply:
I see, yeah that makes sense. Would "Contact form" be acceptable language? This already exists as a translated string, where as "Contact us" does not, far as I'm aware.

Reply:
> We could consider using a hidden h2 for sections when a merchant doesn't include a heading for the section.

Scott, just to clarify for Ken: you are aligned with always including the hidden `h2` on templates _and_ sections if no heading is included?

So instead of: 
```
{%- if section.settings.heading != blank -%}
      <h2 class="title title-wrapper--no-top-margin {{ section.settings.heading_size }}">{{ section.settings.heading | escape }}</h2>
{%- endif -%}
```

We would use something like: 
```
{% assign heading = section.settings.heading | default: "Contact us" %}
 <h2 class="{% if section.settings.heading %}title title-wrapper--no-top-margin {{ section.settings.heading_size }{% else %}visually-hidden{% endif %}">
  {{ heading | escape }}
</h2>
```